plugins/woocommerce/tests/php/src/Internal/StockNotifications/NotificationTests.php

```diff
@@ -0,0 +1,34 @@
+<?php // phpcs:ignore Suin.Classes.PSR4
```

Comment:
I'm curious to know why we disable this rule here.

Reply:
Thanks for noticing; these aren't needed, actually, and it's a leftover from [an issue](https://github.com/suin/phpcs-psr4-sniff/pull/6) I had with my local suin sniff regarding the pre-commit hooks. Removed here https://github.com/woocommerce/woocommerce/pull/57062/commits/b078debd25f0608e62613fdd1e9a7da42eb00597